### PR TITLE
add dogaml.net to Ethereum phishing websites

### PR DESCRIPTION
**Pull request by Rami-Majdoub**
official website: https://dogami.com

### CODE REVIEW

**Comment by Rami-Majdoub** [comment]
Hello @davidwgrossman, I made the requested change.

**Comment by davidwgrossman** [comment]
Hey @Rami-Majdoub can you also provide evidence as to why this is a phishing site?

**Comment by Rami-Majdoub** [comment]
- I got this message from a Discord user (scammer) called MEE6 which is a bot in the server.
scammer message
- the number of minted NFTs on the website is a random number saved to local storage.

File: src/config.json [hunk 1]
```diff
@@ -14261,6 +14261,7 @@
     "racketpool.com",
     "colabration.cc",
     "revokecash.com",
-    "thor.fund"
+    "thor.fund",
+    "dogaml.net",
```
Comment (gravelcycles) [inline src/config.json]:
```suggestion
    "dogaml.net"
```

**Review by gravelcycles — APPROVED** [review]

**Pull request merged**